tests/misc-tests/065-csum-conversion-inject/test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Verify the csum conversion can still resume after an interruption
+
+source "$TEST_TOP/common" || exit
+source "$TEST_TOP/common.convert" || exit
+
+check_experimental_build
+check_injection
+setup_root_helper
+prepare_test_dev
+
+test_resume_data_csum_generation()
+{
+	local new_csum="$1"
+	local tmp=$(_mktemp "csum-convert")
+
+	# Error at the end of the data csum generation.
+	export INJECT="0x4de02239"
+	run_mustfail_stdout "error injection not working" \
+		"$TOP/btrfstune" --csum "$new_csum" "$TEST_DEV" &> $tmp
+	cat "$tmp" >> "$RESULTS"
```

Comment:
I'd like to avoid any use of raw $RESULTS in the test cases, there should be helper for everything (like _log) but here we need a new one (_log_file), this is for a later cleanup.

Reply:
I can do a cleanup later. As the pattern is utilized quite widely.

Reply:
Fortunatelly it's not used widely, the other instance is in misc-tests/005-convert-progress-thread-crash/test.sh, the test infrastructure can use RESULTS freely.

Reply:
I mean not only the `_log_file`, but also some location that can be converted to `_log`.

E.g. all the `tests/*-test.sh` script are using something like `echo "=== START TEST $testname" >> "$RESULTS"`.
The same for a lot of internal usage inside `tests/common`.